game_board.py

```diff
@@ -0,0 +1,199 @@
+import curses
+import os
+import time
+
+# The path of the textfile of the map
+map_path = 'Ascii board.txt'
+
+def read_map(path):
+    try:        
+        board = open( path,'r')
+        board.close 
+
+        board_txt = board.read()
+        return board_txt
+    except IOError:
+        print("Can not find the file" )
+
+
+
+
+def map_cord(str_board):
+    """
+    Reads the string from left to right and appends every char to list with its x any y pos
+    the top of the menu. 
+    Returns a list of lists where every list has [char,xPos,Ypos]
+    Example "X  " => [['X', 0, 0], [' ', 1, 0], [' ', 2, 0]...,]
+
+    Keyword arguments:
+    str_board -- the string of the map
+    """
+ 
+    matrix = [[x for x in line] for line in str_board.split('\n')]
+    map_xy = []
+    y = 0
+    for row in matrix:
+        x = 0
+        for c in row:
+            char_xy = [c,x,y]    
+            map_xy.append(char_xy)
+            x += 1
+        y += 1
+    return map_xy
+
+def move_plus():
+    """
+    Finds all the "+" chars in the map_xy 
+    Returns a list of lists where every list has ["+",xPos,Ypos]
+    Example [['X', 0, 0], [' ', 1, 0],...,['+', '4', '0']] => [['+', '4', '0'],...]
+    """
+    map_xy =map_cord(read_map(map_path))
+    move = []
+    for cord in map_xy:
+        if cord[0] == "+":
+            move.append([cord[0],str(cord[1]),str(cord[2])])
+    return move
+
+def print_map(screen):
+    """Reads the map_xy list  and prints the chars in the Screen
+
+    Keyword arguments:
+    screen -- the curses screen
+    map_xy -- list of lists the chars and their positions 
+    """
+    map_xy = map_cord(read_map(map_path))
+    for cord in map_xy:
+        h,w = screen.getmaxyx()
+        y = cord[2]+ 5
+        x = cord[1] +  w//3    
+        char = cord[0]
+        if char == "O":
+            curses.init_pair(2,curses.COLOR_YELLOW,curses.COLOR_BLACK)
+                        
+            screen.addstr(y,x,char,curses.color_pair(2))
+        elif char == "X": 
+            curses.init_pair(3,curses.COLOR_RED,curses.COLOR_BLACK)
+
+            screen.addstr(y,x,char,curses.color_pair(3))
+        elif char != "+" and char != "O" and char !="X":
```

Comment:
You dont need to check if the char is "O" or "X" since that already has been checked in the previous if statements
```suggestion
        elif char != "+":
```